code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -169,7 +174,17 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 
 	return 1
 
-/datum/gas_mixture/merge(datum/gas_mixture/giver)
+	//Merges all air and volume from giver into self
+	//Returns: 1 if we are mutable, 0 otherwise
+/datum/gas_mixture/proc/full_merge(datum/gas_mixture/giver)
+	if(!merge(giver))
+		return 0
+	volume += giver.volume
+	return 1

```

Comment:
What is the intended use for this proc? Why would you want to merge volume?

Reply:
when i want really merge air conteyners. like pipelines.

Reply:
Shouldn't pipenets do the work for you already? Or do you plan on giving those the rework they desperately need.

Reply:
as i write upper
Now i think about make atmos machinery work more similar to pipes. Now pipeline share gas from self pipe and all connected things by reconcile_air()
i want remove reconcile_air() and make machinery work directly with pipeline air

Reply:
I'd recommend doing that in a separate pr so this one doesn't get slowed down by the idea.

Reply:
now this is garbage of smal air ideas that i push here from work.
i clear it later or separate to new pr's. 
Now it place where someone who understands something here can say something about it.